harmonica/forward/prism_layer.py

```diff
@@ -334,6 +341,9 @@ def gravity(
             Name of the property layer (or ``data_var`` of the
             :class:`xarray.Dataset`) that will be used for the density of each
             prism in the layer. Default to ``"density"``
+        thickness_threshold : int or float
+            Minumum prism thickness needed to be to including in the forward
+            gravity calculation. Default to 0.
```

Comment:
Small edit on the text (and fix typo):

```suggestion
            Minimum prism thickness needed to be included in the forward
            gravity calculation. Prisms thinner that this threshold will be ignored.
            Default to 0.
```